CONTRIBUTING.md

```diff
@@ -0,0 +1,89 @@
+TL;DR:
+- Communication is in [French](https://www.legifrance.gouv.fr/loda/article_lc/LEGIARTI000006421209).
+- Code and code documentation are in [English (BR)](https://en.wikipedia.org/wiki/English_as_a_lingua_franca).
+- Code documentation is in the [JSDoc](https://jsdoc.app/) format.
+- Commits follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/).
+- Code styling is progamatically enforced and follows [Standard](https://standardjs.com/).
+- Testing is tactical and is done with [Vutest](https://vutest.dev) and [Cypress](https://www.cypress.io/).
```

Comment:
Je vois ce que tu veux dire avec "tactical" vu qu'on en a parlé lors du dernier hebdo, mais je suis pas certain que ça soit clair pour un non-initié ?

Reply:
Let's give it a thought, I'm out of ideas on thato ne.

Reply:
On peut mettre "Testing is done with..." pour l'instant et voir plus tard.
Quitte à ajouter une section "testing" dédiée plus tard ?